labels.mss

```diff
@@ -129,166 +129,103 @@
-
-        [zoom >= 16] {
-          shield-file: url("symbols/shields/tertiary_[width]x[height]_z16.svg");
-        }
-        [zoom >= 18] {
-          shield-file: url("symbols/shields/tertiary_[width]x[height]_z18.svg");
-        }
-      }
+      text-name: "[refs]";
+      text-size: 8;
+      text-fill: @road_text;
+      text-spacing: 300;
+      text-clip: false;
+      text-placement: line;
+      text-face-name: @sans;
+      text-halo-radius: @standard-halo-radius;
+      text-halo-fill: @road_halo;
+      text-repeat-distance: @major-highway-text-repeat-distance;
+    }
+    [zoom >= 14] {
+      text-size: 9;
+    }
+    [zoom >= 15] {
+      text-size: 10;
+    }
+    [zoom >= 17] {
+      text-size: 11;
+    }
+    [zoom >= 19] {
+      text-size: 12;
     }
   }
 
+  [highway = 'residential'],
   [highway = 'unclassified'],
-  [highway = 'residential'] {
-    [zoom >= 15] {
+  [highway = 'service'],
+  [highway = 'road'] {
+    [zoom >= 16] {
       text-name: "[refs]";
-      text-size: 8;
-
-      [zoom >= 16] {
-        text-size: 9;
-      }
-      [zoom >= 18] {
-        text-size: 10;
-      }
-
+      text-size: 9;
       text-fill: @road_text;
-      text-face-name: @sans;
+      text-spacing: 300;
+      text-clip: false;
       text-placement: line;
-      text-repeat-distance: @major-highway-text-repeat-distance;
-      text-halo-radius: 2;
+      text-halo-radius: @standard-halo-radius;
       text-halo-fill: @road_halo;
-      text-spacing: 760;
-      text-clip: false;
+      text-face-name: @sans;
+      text-repeat-distance: @minor-highway-text-repeat-distance;
+    }
+    [zoom >= 17] {
+      text-size: 11;
+      text-spacing: 400;
+    }
+    [zoom >= 19] {
+      text-size: 12;
+      text-spacing: 400;
     }
   }
 
-  [highway = 'track'] {
-    [zoom >= 15] {
+  [highway = 'living_street'],
+  [highway = 'pedestrian'] {
+    [zoom >= 16] {
       text-name: "[refs]";
-      text-size: 8;
-      text-dy: 5;
-
-      [zoom >= 16] {
-        text-size: 9;
-        text-dy: 7;
-      }
-      [zoom >= 17] {
-        text-size: 11;
-        text-dy: 9;
-      }
-
+      text-size: 9;
+      text-fill: @road_text;
+      text-spacing: 300;
       text-clip: false;
-      text-fill: #222;
-      text-face-name: @sans;
+      text-placement: line;
       text-halo-radius: @standard-halo-radius;
       text-halo-fill: @road_halo;
-      text-margin: 10;
-      text-placement: line;
-      text-spacing: 760;
-      text-repeat-distance: @major-highway-text-repeat-distance;
-      text-vertical-alignment: middle;
+      text-face-name: @sans;
+      text-repeat-distance: @minor-highway-text-repeat-distance;
+    }
+    [zoom >= 17] {
+      text-size: 11;
+    }
+    [zoom >= 19] {
+      text-size: 12;
     }
   }
 }
 
 #roads-text-name {
   [highway = 'motorway'],
   [highway = 'trunk'],
-  [highway = 'primary'],
-  [highway = 'construction'][construction = 'motorway'],
```

Comment:
Why dropping the name of roads under construction?

Reply:
What’s the point of having them ?

Reply:
We'd lose the name on "Avenue de la République" here for instance https://www.cyclosm.org/#map=19/48.80997/2.30184/cyclosm.

Reply:
The road is not even rendered, the name is on nothing, so it is a good fix :]